src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -257,28 +292,7 @@ export const ShippingRoute: FC<ShippingProps> = props => {
   }
 
   const selectShipping = async (editedAddress?: MutationAddressResponse) => {
-    if (shippingOption === "SHIP") {
-      if (isCreateNewAddress()) {
-        // validate when order is not pickup and the address is new
-        const { errors, hasErrors } = validateAddress(address)
-        const { error, hasError } = validatePhoneNumber(phoneNumber)
```

Comment:
Previously the basic form validation was in `finalizeFulfillment()` which is after address verification. I moved it out to before the address verification.